src/main/java/com/kodilla/ecommercee/domain/Order.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.ecommercee.domain;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity(name="order")
```

Comment:
niepotrzebny parametr name w Entity (jak czegoś nie trzeba to nie ustawiamy tego ;) )

Reply:
zmienione